rlberry/agents/bandits/index_agents.py

```diff
@@ -34,9 +34,8 @@ class IndexAgent(BanditWithSimplePolicy):
 
     name = "IndexAgent"
 
-    def __init__(self, env, index_function=lambda rew, t: np.mean(rew), **kwargs):
+    def __init__(self, env, index_function=None, **kwargs):
         BanditWithSimplePolicy.__init__(self, env, **kwargs)
-        self.n_arms = self.env.action_space.n
```

Comment:
If you rename all the "index" to "indices", shouldn't it be "indices_function" and "Indice_Agent" ?

Reply:
Ok, maybe I should read the Oxford dictionary some more ;)